perception/multi_object_tracker/src/tracker/model/bicycle_tracker.cpp

```diff
@@ -137,15 +143,28 @@ BicycleTracker::BicycleTracker(
   ekf_.init(X, P);
 
   // Set lf, lr
-  double point_ratio = 0.2;  // under steered if smaller than 0.5
-  lf_ = bounding_box_.length * point_ratio;
-  lr_ = bounding_box_.length * (1.0 - point_ratio);
+  lf_ = bounding_box_.length * 0.3;  // 30% front from the center
+  lr_ = bounding_box_.length * 0.3;  // 30% rear from the center
 }
 
 bool BicycleTracker::predict(const rclcpp::Time & time)
 {
   const double dt = (time - last_update_time_).seconds();
-  bool ret = predict(dt, ekf_);
+  if (dt < 0.0) {
+    RCLCPP_WARN(logger_, "dt is negative. (%f)", dt);
+    return false;
+  }
+  // if dt is too large, shorten dt and repeat prediction
+  const double dt_max = 0.11;
```

Comment:
```suggestion
  constexpr double dt_max = 0.11;
```

Are there any reason that this value is not `0.1`? (Just curious)

Reply:
(expecting prediction process frequency is 10hz, time interval of 0.1 second)
It can be expected that the interval of the tracker is fluctuating.

This modification (repeat the prediction) can cause extra calculation and shouldn't be used frequently.
If the maximum is set to exactly expected time interval `0.1`, it can trigger this extra calculation in some chances.